src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1587,15 +1592,13 @@ public String getFeedbackSessionResultSummaryInSectionAsCsv(String courseId,
      * Preconditions: <br>
      * * All parameters are non-null.
      */
-    public boolean hasStudentSubmittedFeedback(String courseId, String feedbackSessionName, String studentEmail)
+    public boolean hasStudentSubmittedFeedback(FeedbackSessionAttributes fsa, String studentEmail)
             throws InvalidParametersException, EntityDoesNotExistException {
```

Comment:
Is this change related to this PR? If not, should do as a separate issue. On a related note, passing an entire object is not necessarily better.  It increases coupling unnecessarily. A method should not receive more than it needs.